docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,11 @@
+### `cosmos/cosmos-sdk` Release Process
+
+- [ ] 1. Decide on release designation (are we doing a patch, or minor version bump) and start a P.R. for the release
+- [ ] 2. Add commits/PRs that are desired for this release **that haven’t already been added to develop**
+- [ ] 3. Ensure that `CHANGELOG.md` contains links to issues/PRs for each item
```

Comment:
This should be in the PR checklist, not the release checklist (the PR should reference the issues in the changelog entries to begin with)

Reply:
I think this should be in the PR checklist as well, but we need to do a `CHANGELOG.md` check and clean during the release process as well. Maybe I combine #3 and #4?

Reply:
Also `PENDING.md` needs to be moved to `CHANGELOG.md` - combining 3/4 sounds good.

Reply:
`PENDING.md`?

Reply:
We now use `PENDING.md` for the pending changelog entries to avoid confusion for older PRs.

Reply:
Addressed